src/functional/ffront/foast_passes/type_deduction.py

```diff
@@ -126,10 +36,10 @@ def are_broadcast_compatible(left: TypeInfo, right: TypeInfo) -> bool:
 
     Examples:
     ---------
-    >>> int_scalar_t = TypeInfo(common_types.ScalarType(kind=common_types.ScalarKind.INT64))
+    >>> int_scalar_t = TypeInfo(ScalarType(kind=ScalarKind.INT64))
```

Comment:
For modules containing a set of similar objects I always prefix with modules. If you change this here we should at least do it consistently everywhere in a separate PR.

Reply:
Is it acceptable to follow the convention, but for example `import common_types as ct` to improve readability?

Reply:
Yes this makes sense, I think we should use `ct` for `common_types` everywhere then.